CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ Todos los cambios notables a este proyecto serán documentados en este archivo.
 El formato está basado en [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 y este proyecto adhiere a [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.4.2] - 2018-09-03
```

Comment:
If we are also making a release after this PR lands on master, then you need to change the version in `package.json` and `Onepay.js` too (and remember to run `npm run build`).

Reply:
There are currently changed